tmail-backend/mailbox/encrypted/api/src/main/scala/com/linagora/tmail/encrypted/EncryptedMailboxManager.scala

```diff
@@ -124,7 +125,7 @@ class EncryptedMailboxManager @Inject()(mailboxManager: MailboxManager,
 
   override def logout(session: MailboxSession): Unit = mailboxManager.logout(session)
 
-  override def getMailboxReactive(mailboxId: MailboxId, session: MailboxSession): Publisher[MessageManager] = mailboxManager.getMailboxReactive(mailboxId, session)
+  override def getMailboxReactive(mailboxId: MailboxId, session: MailboxSession): Publisher[MessageManager] = SMono.fromCallable(() => getMailbox(mailboxId, session))
```

Comment:
That's not reactive compliant.

Please do a non blocking implementation here.

Reply:
I don't get what's wrong with the current implementation?

Reply:
getMailbox blocks.

In a reactive thread.

It's a major performance breakdown.